cmd/nigiri/resources/docker-compose.yml

```diff
@@ -234,29 +234,43 @@ services:
       - ./volumes/lightningd:/.lightning
       - ./volumes/bitcoin:/etc/bitcoin
 
+  ark-wallet:
+    container_name: ark-wallet
+    image: ghcr.io/arkade-os/arkd-wallet:v0.7.0
+    depends_on:
+      - bitcoin
+    environment:
+      ARKD_WALLET_ESPLORA_URL: "http://chopsticks:3000"
+      ARKD_WALLET_NEUTRINO_PEER: "bitcoin:18444"
```

Comment:
Why not leveraging JSON RPC Bitcoin core so we keep that tested as favorite wallet backend for now?

Reply:
done 
ade0b84